cmd/rpcdaemon/health/check_time.go

```diff
@@ -26,6 +26,8 @@ func checkTime(
 	if ts, ok := i["timestamp"]; ok {
 		if cs, ok := ts.(hexutil.Uint64); ok {
 			timestamp = int(uint64(cs))
+		} else if cs, ok := ts.(uint64); ok {
+			timestamp = int(cs)
 		}
```

Comment:
have a fatal (in separate `else` condition) in case `ts` can't be casted to the two types.

Reply:
> have a fatal (in separate `else` condition) in case `ts` can't be casted to the two types.

@sudeepdino008 could you please create a PR with this safeguard